test/pummel/test-stream-pipe-multi.js

```diff
@@ -49,10 +49,6 @@ process.on('exit', function() {
   assert.equal(cnt, rclosed, 'readable streams closed');
 });
 
-for (var i = 0; i < chunkSize; i++) {
-  chunkSize[i] = i % 256;
-}
-
```

Comment:
Are you sure this is safe to remove? Obviously the `chunkSize[i]` part is wrong, but perhaps this should be kept and just replace `chunkSize[i]` with `data[i]` assuming that was the original intention?